egograph/pipelines/tests/unit/youtube/test_extraction.py

```
@@ -0,0 +1,80 @@
+            url="https://www.youtube.com/watch?v=video-3",
+        ),
+        _row(page_view_id="pv-reload-2", transition="reload"),
+    ]
+
+    result = extract_youtube_watch_events(rows)
+
+    assert [event["source_event_id"] for event in result] == ["pv-link-1", "pv-link-2"]
+
+
+def test_no_transition_field_is_kept():
+    """page_view に transition が無い場合（後方互換）は除外しない。"""
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)